### DISCUSSION TITLE
Using manually configured PID parameters?

### DISCUSSION THREAD

**Discussion by mairas** [ScratMan/HASmartThermostat#175] [Q&A] [answered]
Another newbie question: it seems the Smart Thermostat is now storing the PID parameter state in HA's DB storage and ignores the manually set values after the first run. Or at least it seems like that's how it's working for me. This is a bit confusing; is it possible to have the configuration.yaml settings override the DB storage?

**Comment by Chupaka** [comment, marked as answer]
Use "smart_thermostat.set_pid_gain" service in Home Assistant (e.g. via Developer tools) to update coefficients of existing thermostat

**Reply by mairas** [reply]
Thanks, figured that out, too. It just wasn't very clearly told in the README.
